cli/src/main/java/com/box/l10n/mojito/cli/command/ThirdPartySyncActionsConverter.java

```diff
@@ -1,15 +1,15 @@
 package com.box.l10n.mojito.cli.command;
 
-import com.box.l10n.mojito.rest.client.ThirdPartySync;
+import com.box.l10n.mojito.rest.ThirdPartySyncAction;
```

Comment:
oh sorry, maybe I wasn't super clear last time we chatted about it. I don't want the rest client and the webservice layer to be linked closely. The main reason is that in most cases there is not a 1:1 mapping between what's done in the server and what makes sense in the client (hibernate entities, slightly different object). I know that in many case the POJO are or file like duplicates between the client and server but I'd rather keep that way. Let's not have the rest client as dependency of webapp, it also becomes messy and we may end up having some code going through the client instead of directly through the service which doesn't make sense. It may become confusing.

Reply:
Just updated the code